chart/templates/platform-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if include "vcluster.rbac.createPlatformSecretRole" . }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "vcluster.rbac.platformRoleName" . }}
+  namespace: {{ include "vcluster.rbac.platformSecretNamespace" .}}
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "watch", "patch"]
```

Comment:
Why patch here?

Reply:
Because of this Patch here: https://github.com/loft-sh/vcluster/blob/4a32dc7bc4619ca17239d596b17882cca5e048c9/pkg/platform/secret.go#L89

Reply:
But thats part of the cli package, so we don't need that "server side"